src/crawler.ts

```diff
@@ -878,7 +878,7 @@ self.__bx_behaviors.selectMainBehavior();
       try {
         const { fetched, mime, ts } = await timedRun(
           directFetchCapture({ url, headers: this.headers, cdp }),
-          FETCH_TIMEOUT_SECS,
+          this.params.timeout,
```

Comment:
As Benoit said, I think maybe better to rename this to `this.params.pageLoadTimeout` to avoid confusion with the alias and other timeout values we're using.